packages/@okta/i18n/src/properties/login.properties

```diff
@@ -642,6 +642,8 @@ registration.default.callbackhook.error=We could not process your registration a
 registration.error.userName.invalidEmail=Invalid email address
 registration.error.password.passwordRequirementsNotMet=Password requirements were not met
 registration.error.userName.notUniqueWithinOrg=An account with that email already exists
+registration.model.validation.field.string.too.short = Please enter something at least {0} characters long
```

Comment:
@pradeepdewda-okta @sherizada-okta 
feels like we really show update Model props and validation function (https://github.com/okta/okta-ui/blob/master/packages/courage/src/framework/Model.js#L304)
or even change Courage (there was discussion on this but didn't find it now)